project/Build.scala

```diff
@@ -185,8 +181,8 @@ object ChillBuild extends Build {
     autoScalaLibrary := false,
     libraryDependencies ++= Seq(
       "org.apache.hadoop" % "hadoop-core" % "0.20.2" % "provided",
-      "org.slf4j" % "slf4j-api" % "1.6.6",
-      "org.slf4j" % "slf4j-log4j12" % "1.6.6" % "provided"
+      "org.slf4j" % "slf4j-api" % "1.7.12",
```

Comment:
why are you bumping these up? Do we need it? This going to force it on everyone downstream (which includes scalding, spark, summingbird, and a lot of people internal at Twitter).